packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -216,6 +216,9 @@ async function nodeImport(
   ) => {
     const resolved = tryNodeResolve(id, importer, options, false)
     if (!resolved) {
+      if (path.isAbsolute(id)) {
+        return id
+      }
```

Comment:
Attempt: if `tryNodeResolve` returns `undefined` but our id is an absolute path, it might be fully resolved already. Just return it.

Reply:
Any reason we shouldn't check for absolute path before the `tryNodeResolve` call?